DiegoFlores/Day2/Challenge/Main.java

```diff
@@ -0,0 +1,47 @@
+package person;
+
+    	public static final String STUDENT1CAREER = "architecture";
```

Comment:
For the final name variables the words will need to by separeted by "_" example: STUDENT_ONE_CARRER, fix all the names